manage.py

```diff
@@ -18,8 +18,8 @@
 manager.add_option(
     '-m', '--cfgmode', dest='config_mode', default='Development')
 manager.add_option('-f', '--cfgfile', dest='config_file', type=p.abspath)
-manager.add_command('runserver', Server(port=c.PORT))
-manager.add_command('serve', Server(port=c.PORT))
+manager.add_command('runserver', Server(host=c.HOST, port=c.PORT))
+manager.add_command('serve', Server(host=c.HOST, port=c.PORT))
 manager.main = manager.run
```

Comment:
For some reason, this is what worked. Let me know if there is a better solution.